concent-builder/build-test-and-push.yml

```diff
@@ -85,6 +85,7 @@
       always:
         - name:  Stop postgres container
           shell: docker stop {{ cluster }}-{{ image_prefix }}postgresql-unittest
+          ignore_errors: yes
```

Comment:
If it's needed here, don't we need it for `docker network rm` as well?

Reply:
If there was another task after `docker network rm`, we would also need `ignore_errors` to perform it.